.circleci/config.yml

```diff
@@ -169,7 +170,7 @@ jobs:
           export AGENT_BIN=$(pwd)/bundle/bin/signalfx-agent
           export BUNDLE_DIR=$(pwd)/bundle
           export TEST_SERVICES_DIR=$(pwd)/test-services
-          sudo -E pytest \
+          sudo -E $PYENV_ROOT/shims/pytest \
```

Comment:
Why do we need this??

Reply:
`$PYENV_ROOT/shims/` is auto added to PATH for the circleci user by pyenv, but sudo protects PATH.